src/win/tty.c

```diff
@@ -2384,3 +2379,17 @@ static void uv__tty_console_signal_resize(void) {
     uv_mutex_unlock(&uv__tty_console_resize_mutex);
   }
 }
+
+int uv_tty_set_vterm_state(uv_tty_vtermstate_t state) {
+  uv_sem_wait(&uv_tty_output_lock);
+  uv__vterm_state = state;
+  uv_sem_post(&uv_tty_output_lock);
+  return 0;
+}
+
+int uv_tty_get_vterm_state(uv_tty_vtermstate_t* state) {
```

Comment:
could return `uv_tty_vtermstate_t` ?

Reply:
In that case, the problem is what to return on a Unix system. We can also add an enumerator that indicates an error to `uv_tty_vterm_state_t`, but in that case the behavior of passing it to `uv_tty_set_vterm_state()` is a problem. I made the current implementation for these reasons.